client/src/common/utils/helpers.js

```diff
@@ -44,9 +44,11 @@ const fromEntries = convertedArray =>
 export const asyncTypes = key =>
   asyncPostfixes.map(postfix => [key, postfix].join('_'));
 
-export const enumerable = namespace => (...keys) =>
+export const enumerable = (namespace = null) => (...keys) =>
```

Comment:
It would be nice to add some jsDoc with example to this function cause it is pretty hard to understand

https://devdocs.io/jsdoc/howto-es2015-modules
I did something like this in some places in project so you can copy the pattern